### DISCUSSION TITLE
Add comment column

### DISCUSSION THREAD

**Discussion by lbschenkel** [probonopd/irdb#180] [General]
Hi. I think it would be useful to allow an optional `comment` (or `notes`) column, just after `function`, which would enable documenting a little bit better what a specific code does.

For example, a Panasonic code for `INPUT TV` could be documented as "Switches input to TV, or toggles tuner among analog/digital/etc.".

I have documented/clarified a number of codes over the years and I would like to be able to contribute this documentation to this project. What do you think? If you like this idea, I can then submit some PRs.

**Comment by probonopd** [comment]
Wdyt @bengtmartensson?

**Comment by bengtmartensson** [comment]
The current (CSV) format has a number of shortcomings; the absence of a comment field is just one. I have written it down  as #58, together with an alternative (Girr). This would be a systematic solution.
